Fix the Makefile.

N.B. The upstream CVS HEAD has been converted to GNU autotools, but a
tarball has not yet been released. When the next release tarball is made
available, this patch can be removed.

Signed-off-by: Simon Dawson <spdawson@gmail.com>

diff -Nurp a/Makefile b/Makefile
--- a/Makefile	2012-01-17 22:49:01.000000000 +0000
+++ b/Makefile	2013-05-04 22:29:19.947073948 +0100
@@ -64,7 +64,7 @@ install-conf:
 	$(INSTALL) -d $(ETCINSTALLDIR)
 	$(INSTALL) --mode=0644 minidlna.conf $(ETCINSTALLDIR)
 
-minidlna:	$(BASEOBJS) $(LNXOBJS) $(LIBS)
+minidlna:	$(BASEOBJS) $(LNXOBJS)
 	@echo Linking $@
 	@$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $(BASEOBJS) $(LNXOBJS) $(LIBS)
 
